src/euidSdk.ts

```diff
@@ -69,6 +70,12 @@ export function __euidInternalHandleScriptLoad() {
   const callbackContainer: CallbackContainer = {};
   window.__euid = new EUID(callbacks, callbackContainer);
   if (callbackContainer.callback) callbackContainer.callback();
+  if (window.__euid instanceof EUID) {
```

Comment:
Does this new code possibly deserve it's own function? it might make it clearer, given the important line is nested two levels deep.

Reply:
Yeah I like that!